integrations/terraform/Makefile

```diff
@@ -46,7 +46,7 @@ $(BUILDDIR)/terraform-provider-teleport_%: terraform-provider-teleport-v$(VERSIO
 
 CUSTOM_IMPORTS_TMP_DIR ?= /tmp/protoc-gen-terraform/custom-imports
 # This version must match the version installed by .github/workflows/lint.yaml
-PROTOC_GEN_TERRAFORM_VERSION ?= v2.2.0
+PROTOC_GEN_TERRAFORM_VERSION ?= v3.0.0
```

Comment:
@hugoShaka I wasn't able to download `github.com/gravitational/protoc-gen-terraform@v2.2.0` so I updated this to v3.0.0. Is this correct or is there a way for me to get v2.2.0 still?

Reply:
I think this just got fixed on master, sorry for the ping.

Reply:
Yeah you should re-render with the old version. The TF codebase is not ready for 3.0.0, the code from the PR will not compile